Core/LocalAdmin.cs

```diff
@@ -99,10 +99,12 @@ internal enum HeartbeatStatus : byte
 
     internal LocalAdmin()
     {
-        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)){
```

Comment:
We don't do same line `{`.

Reply:
This is being removed as it is a 1 liner.